.pre-commit-config.yaml

```diff
@@ -46,3 +46,11 @@ repos:
     -   id: doc8
         additional_dependencies: [pygments]
         args: [--quiet,--ignore-path=Doc/examples/ec_*.txt]
+-   repo: local
+    hooks:
+    -   id: doi-link-style
+        name: Enforce https://doi.org/ style
+        description: Check DOI link style, see https://www.crossref.org/display-guidelines/
+        entry: '(?i)(doi:|dx\.doi\.org|http://doi\.org)'
+        language: pygrep
+        files: \.(rst|tex)$
```

Comment:
What about `.py` files? I see you changed `Bio/__init__.py` but the extension is absent from the check. Is this OK?

Reply:
I tried that, and we get a false positive from a database cross-reference in the tests. The easiest option was follow the TravisCI/Tox setup, and only check ``*.rst`` and ``*.tex``  - I didn't immediately see a nice alternative.

Reply:
👍